cmdproc/wordpic.py

```diff
@@ -0,0 +1,116 @@
+import random
+from json import load
+from pathlib import Path
+
+from config import ENV
+from telegram import (BotCommand, InlineKeyboardButton, InlineKeyboardMarkup,
+                      Update)
+from telegram.ext import CallbackContext, CallbackQueryHandler, CommandHandler
+from utils.fileproc import gen_pic_dict_from_csv
+from utils.filters import check_chatid_filter
+
+word_dict = {}
+chapter_dict = {}
+
+
+def check_extra_dict(dict_dir):
+    # 检查是否有用户自定义的单词库
+    if dict_dir is None:
+        return 0
+    try:
+        with open(f"{dict_dir}/res/picwords.csv", 'r') as csvfile:
+            word_dict, chapter_dict = gen_pic_dict_from_csv(csvfile)
+            print(f"看图识词单词条目：{len(word_dict)}个")
+            return len(word_dict)
+    except FileNotFoundError:
+        return 0
+
+
+def reload_dict():
+    global word_dict
+    global chapter_dict
+    # 加载内置单词库
+    with open('pic_dict.json', 'r') as wd:
+        word_dict = load(wd)
+    with open('chapter_dict.json', 'r') as wd:
+        chapter_dict = load(wd)
+
+    # 加载用户自定义单词库
+    try:
+        with open(f"{ENV.DATA_DIR}/res/picwords.csv", 'r') as csvfile:
+            word_dict, chapter_dict = gen_pic_dict_from_csv(
+                csvfile, word_dict, chapter_dict)
+    except FileNotFoundError:
+        pass
+
+
+reload_dict()
+
+again = InlineKeyboardMarkup([
+    [InlineKeyboardButton("🎲 Play again 🕹", callback_data=f"getnew:mm"),
+     InlineKeyboardButton("🧑🏻‍🏫 🗣Help 👩🏻‍🏫", callback_data=f"getpron:")
+     ]])
+
+
+def check_answer(question, answer, filenumber):
+    # 问题的答案是否正确
+    # question : 图中的单词
+    # answer : 用户回答的号码
+    # filenumber : 图片的页数编号
+    if question.lower() in word_dict:
+        words = word_dict[question.lower()]
+        for word in words:
+            if answer == word["number"] and f"{filenumber}.jpg" == word["filename"]:
+                return True
+    return False
+
+
+def mm_command(update: Update, context: CallbackContext) -> None:
```

Comment:
The function name can be more meaningful?

Reply:
> 

haha, maybe map_word_to_pic is better

Reply:
:+1: